src/liballoc/tests/lib.rs

```diff
@@ -36,8 +36,8 @@ fn hash<T: Hash>(t: &T) -> u64 {
     s.finish()
 }
 
-// FIXME: Instantiated functions with i128 in the signature is not supported in Emscripten.
-// See https://github.com/kripken/emscripten-fastcomp/issues/169
+// FIXME: instantiated functions with `i128` in the signature is not supported in Emscripten.
+// See issue #169.
```

Comment:
This change makes it look like this is an issue on the `rust-lang/rust` repo, which it is not.

Reply:
Oops, my bad.

Reply:
Fixed now, thanks!